core/quill.js

```diff
@@ -445,6 +449,10 @@ class Quill {
       true,
     );
   }
+
+  startFormat() {
```

Comment:
I shared my thoughts in the related PR

Reply:
refactored. at present, some additional calls special methods not needed